doc/global-banner.md

```diff
@@ -11,6 +11,23 @@ In `app/views/notifications/_global_bar.html.erb`
 2. Update the `show_global_bar` variable to `true`
 3. Deploy static
 
-The usual rules apply with static template caching.
-
 ![screenshot](/doc/global-banner.png?raw=true)
+
+If you are in the production environment, once the origin cache is purged the CDN cache will be purged automatically.
+This will clear cache for the top 10 most visited pages.
+
+### The banner is not showing / not clearing!
+Usually this is because the caching has not cleared properly. This can be at various points in our stack as well as locally in your browser. Things to try:
+
+* Make sure you are actually looking at a page on the environment you released the banner. Remember to use the equivalent page for the environment (often staging) on which you are testing/releasing the banner.
+* Test the page with curl to circumvent any browser-based caching. Chrome seems to aggressively cache on occasion. You can also test in a private browser instance.
+* Clear the caches using these Rake tasks
+  - [memcache](https://deploy.integration.publishing.service.gov.uk/job/clear-frontend-memcache/)
```

Comment:
I wonder if we should link to the staging and production version of this URL as well, in a similar way to https://docs.publishing.service.gov.uk/manual/emergency-publishing.html#2-deploy-the-banner-using-jenkins.